test/serverTest.js

```diff
@@ -159,6 +159,7 @@ describe('Brave Sensor server', () => {
             let sessions = await db.getAllSessionsFromLocation(testLocation1Id)
             expect(sessions.length).to.equal(1)
             expect(sessions[0].od_flag).to.equal(1)
+            await sleep(1000)
```

Comment:
What are the new `sleep` s for?

Reply:
my tests hang locally exactly in between these two, and if solving w a sleep i figured it's preferable to be one-offs rather than in the before/after eachs

Reply:
OK, that's too bad, but if it works, then it works :+1: